FRPs/mempool-privacy-frp.md

```diff
@@ -0,0 +1,33 @@
+
+---
+id: TBD
+title: Mempool Privacy
+team: HashCloak
+created: 2021-02-24
+paper: 1
+---
+
+# Mempool Privacy
+
+## Abstract
+
+A feature that is missing from MEV-Geth is complete privacy. That is, the ability to keep incoming transactions in the mempool private. As a result, certain kinds of censorship and ordering attacks are still possible despite MEV-Geth's ability to provide pre-trade and failed trade privacy.
+
+The goal of this RFP is to survey privacy solutions that can be used to provide MEV-Geth with mempool privacy and attempt to start formalizing the problem of mempool privacy for L1 and L2 blockchain constructions.
+
+## Plan and Deliverables
+- Define a set of properties of a desirable mempool privacy solution
+    - Identify what may be different for different L1 constructions
```

Comment:
by this you mean different consensus algorithms? different mempool architectures? check the ongoing FRP for related work: https://github.com/flashbots/mev-research/pull/27.

Reply:
Yes, I do mean different mempool architectures. Generally, this is more of an implementation detail depending on the L1. So, I just broadly put it as different L1 constructions.